src/frontend/src/components/modals/LicenseModal.tsx

```diff
@@ -0,0 +1,97 @@
+import { Trans, t } from '@lingui/macro';
+import {
+  Accordion,
+  Alert,
+  Divider,
+  Group,
+  LoadingOverlay,
+  Space,
+  Stack,
+  Tabs,
+  Text
+} from '@mantine/core';
+import { ContextModalProps } from '@mantine/modals';
+import { useQuery } from '@tanstack/react-query';
+
+import { api } from '../../App';
+import { ApiEndpoints } from '../../enums/ApiEndpoints';
+import { apiUrl } from '../../states/ApiState';
+import { StylishText } from '../items/StylishText';
+
+export function LicenceView(entries: any[]) {
+  return (
+    <Stack spacing="xs">
+      <Divider />
+      {entries?.length && (
+        <Accordion variant="contained" defaultValue="-">
+          {entries?.map((entry: any, index: number) => (
+            <Accordion.Item key={index} value={`entry-${index}`}>
+              <Accordion.Control>
+                <Group position="apart" grow>
+                  <Text>{entry.name}</Text>
+                  <Text>{entry.license}</Text>
+                  <Space />
+                  <Text>{entry.version}</Text>
+                </Group>
+              </Accordion.Control>
+              <Accordion.Panel>
```

Comment:
Make the license text wrapping on `\n`:
```suggestion
              <Accordion.Panel style={{ whiteSpace: "pre-line" }}>
```